app/templates/index.html

```diff
@@ -12,153 +12,198 @@ <h1 class="govuk-heading-xl">Admin</h1>
 </div>
 <div class="govuk-grid-row">
   <div class="govuk-grid-column-two-thirds">
-    <div class="dmspeak">
-      {% if current_user.has_any_role('admin-ccs-data-controller') %}
-        {# ADMIN CCS DATA CONTROLLER #}
-        <h3 class="heading-xmedium">Search for users</h3>
-        <ul class="govuk-list">
-          <li>
-            <a class="govuk-link" href="{{ url_for('.find_user_by_email_address') }}">Find a user by email</a>
-          </li>
-        </ul>
+    {% if current_user.has_role('admin-ccs-data-controller') %}
+      {# ADMIN CCS DATA CONTROLLER #}
+      <h2 class="govuk-heading-m">Search for users</h3>
```

Comment:
Mismatched elements here - `<h2>` vs `</h3>`

Reply:
(There are a couple of these)

Reply:
Good spot, thank you

Reply:
Fab 👍 Once we have the extra VR scenarios ready to go I'll approve